docs/source/user_guide/faq.rst

```diff
@@ -0,0 +1,43 @@
+fastplotlib FAQ
+===============
+
+.. dropdown:: What is `fastplotlib`?
+
+    `fastplotlib` is a scientific plotting library built on top of the `pygfx <https://github.com/pygfx/pygfx>`_ rendering engine
+    that leverages new graphics APIs and modern GPU hardware to create fast and interactive visualizations.
+
+
+.. dropdown:: What can I do with `fastplotlib`?
+
+    `fastplotlib` allows for:
+        - interactive visualization via an intuitive and expressive API
+        - rapid prototyping and algorithm design
+        - easy exploration and rendering of large-scale data
+
+.. dropdown:: How does `fastplotlib` relate to `matplotlib`?
+
+    `fastplotlib` is **NOT** related to `matplotlib` in any way.
+
+    These are two completely different libraries with their own APIs and use-cases. The `fastplotlib` library is primarily for *interactive*
+    visualization that runs on the GPU using WGPU.
```

Comment:
add something along the lines  of "the fastplotlib way of doing things is different from matplotlib. Using fastplotlib is actually more similar to using numpy! See the "How can I learn to use `fastplotlib` section below"

Reply:
I'm trying to figure out how to stop people from search the docs for "imshow" XD